build.gradle

```diff
@@ -55,13 +55,14 @@ task downloadJdk( type: Download ) {
         !isGenericBuild()
     }
 
-    dest "$downloadJdkDir/"
     overwrite false
     onlyIfModified true
 
+    src getJdkDownloadUrl()
+    dest new File( downloadJdkDir, jdkPackageFile )
```

Comment:
Does this work? Why would other code use `getJdkPackageFile()` instead?

Reply:
It sure does. I guess it was done to explicitly point that this extra property was declared and assigned elsewhere (in `jar.gradle`). `downloadJdkDir` is also an `ext` property, but it is declared in this file and accessed directly.